### PR TITLE
Improvement: Remove vote by '-' icon

### PR DESCRIPTION
**Pull request by Quasilius-Starlord**
## Description

issue: #2913 
A '-' icon to appear when you hover over the "remove vote" option on a note.

## Changelog

Frontend
- SVG icon for remove vote.
- Additional prop and changed type for `DotButton`.
- state to detect mouse hover.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

## (Optional) Visual Changes

- Before hover
before_hover
- On hover over remove
after_hover

### CODE REVIEW

**Comment by blxckOxymoron** [comment]
I think this could be handled using css only
Also, the minus sign is slightly wider than the plus

**Comment by Quasilius-Starlord** [comment]
> Thank you for your PR @Quasilius-Starlord! Are you planning on implementing the suggestions made by @blxckOxymoron? In my personal opinion, doing it your way with an svg is fine, but yeah, it would be good if it would be the same width as the + sign.

Yeah, I'll be implementing as suggested.

**Comment by Quasilius-Starlord** [comment]
- After I made my new changes the UI looks like following: 

Screenshot 2023-10-15 183636

File: src/components/Votes/VoteButtons/RemoveVoteButton.scss [hunk 1]
```diff
@@ -24,12 +26,22 @@
 .vote-button-remove:enabled:hover {
   background-color: var(--accent-color);
   color: $color-white;
+  .vote-button-remove-count {
+    display: none;
+  }
+  .vote-button-remove-icon {
```
[the remark this answers is not in the document]
Comment (Quasilius-Starlord) [reply src/components/Votes/VoteButtons/RemoveVoteButton.scss]: I've made a commit with changes as suggested.

**Review by Resaki1 — APPROVED** [review]
Thank you for your contribution @Quasilius-Starlord!

**Pull request merged**